lib/pub.dart

```diff
@@ -0,0 +1 @@
+export 'package:pub/src/command_runner.dart';
```

Comment:
nit: you could just import this instead, so that there is still nothing exposed by the package

Reply:
hmm, this was covered in https://github.com/dart-lang/pub/pull/1953, not sure why it's in this diff.

will change that to an import in a new PR